### DISCUSSION TITLE
Drop ENUM Knex

### DISCUSSION THREAD

**Discussion by Hakamate** [adonisjs/core#1495] [Help]
Hello everyone,

How can i drop Postgre Type (ENUM) with knex migration Down ?

**Comment by RomainLanz** [comment]
Hey @Hakamate! 👋 

What have you tried so far?

**Reply by Hakamate** [reply]
Hello @RomainLanz 

Drop table, drop constraint (all as enum)

But i don't find anything else

**Comment by skrenek** [comment]
In v4, I've typically resorted to scheduling a raw query to drop the type from pg.  I haven't had the opportunity to do the same in v5 yet, so I don't know if there are any differences.  Here's an example from one of my projects.

    class UpdateTenantsSchema extends Schema {
      up () {
        this.table('tenants', (table) => {
          table.enu('status', ['active', 'inactive'], { useNative: true, enumName: 'tenant_status'}).notNullable().defaultTo('active')
        })
      }
    
      down () {
        this.table('tenants', (table) => {
          table.dropColumn('status')
        })

        this.schedule(async (trx) => {
          await trx.raw(`drop type tenant_status`)
        })
      }
    }

**Comment by thetutlage** [comment]
Yes. Using a raw sql statement is the way to go
